tools/Uno.Sdk.Updater/Program.cs

```diff
@@ -0,0 +1,179 @@
+﻿using System.IO.Compression;
+using System.Text;
+using System.Text.Json;
+using Uno.Sdk.Models;
+using Uno.Sdk.Services;
+using Uno.Sdk.Updater;
+
+const string UnoSdkPackageId = "Uno.Sdk";
```

Comment:
This will need to change to `Uno.Sdk.Private` but is blocked by unoplatform/uno#16848